app/lib/ui/views/active/atr_editing_screen.dart

```diff
@@ -185,6 +185,46 @@ class _ATREditingScreenState extends State<ATREditingScreen> {
                         style: TextStyle(
                             color: NavyBlue, fontWeight: FontWeight.bold),
                       ),
+                      actions: [
+                        IconButton(
+                            icon: Icon(Icons.delete),
```

Comment:
I'm wondering too if this should be a text "Delete" button, like what we do in `PdfScreen` with the words `Email`.

Yeah, actually it's better to have a big red button for the "danger zone"

- [ ] Make this trash icon a Big Red "Delete" text button